### PR TITLE
Add activity exemptions to daylight-off

### PR DESCRIPTION
**Pull request by RyanSweetD2L**
- Add support for building web-components to the daylight-off branch
  using the package.json logic in master
- Move src/*.js to js/* to keep daylight-off more aligned with master

### CODE REVIEW

File: bsi.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<script src="js/page-loading/web-components-ready.js"></script>
+
+<link rel="import" href="bower_components/polymer/polymer.html">
+<link rel="import" href="bower_components/d2l-fastdom-import/fastdom.html">
```
Comment (RyanSweetD2L) [inline bsi.html]: Admittedly, I'm not sure this is required.  I kept it because `master` uses it

Comment (dlockhart) [reply bsi.html]: I've been wanting to make fastdom available to pre-Daylight for a while, so this is very welcome!

**Comment by RyanSweetD2L** [comment]
Most of the new files were shamelessly copied from the `master` branch

**Comment by dlockhart** [comment]
@RyanSweetD2L looks like the Node version for this branch is stuck on `0.12` and you'll need to go up to at least 7 like `master`.

**Comment by RyanSweetD2L** [comment]
I think the build error
> Error: ENOENT: no such file or directory, open '/home/travis/build/Brightspace/brightspace-integration/bower_components/iron-ajax/iron-ajax.html'
is related to activity-exemptions missing an explicit `iron-ajax` reference.  I am adding that to `d2l-activity-exemptions`, creating a new release and bumping this version.

**Pull request merged**